rsc/src/main/scala/rsc/scalasig/Pickle.scala

```diff
@@ -842,7 +852,8 @@ class Pickle private (settings: Settings, mtab: Mtab, sroot1: String, sroot2: St
     def sannots: List[s.Annotation] = {
       sinfo.annotations.toList
     }
-    def ssig: Sig = {
+    def ssig: Sig = cacheSsig.computeIfAbsent(ssym, _ssig)
+    def _ssig(ssym: String): Sig = {
```

Comment:
There is inconsistent privacy of the _ prefixed methods.  How about having a single method and moving cacheSsig.computeIfAbsent into the method body?